client/js/app.modules/header/dw-header-block.tpl.html

```diff
@@ -0,0 +1,11 @@
+<header ng-controller="HeaderCtrl as HeaderCtrl" class="header page__header">
```

Comment:
Not sure that templates should use directive prefix (my concern that templates could be shared). Did you saw it in some style guide?